### PR TITLE
msi: show td-agent and fluentd version

### PR DESCRIPTION
**Pull request by kenhys**
Example:

  > .\td-agent.bat --version
  td-agent 4.0.1 fluentd 1.11.2 (2f80f6cf0a4aa44b9795cd6a66894a6b8fa70d06)

### CODE REVIEW

**Comment by kenhys** [comment]
verified like this:

image

**Comment by cosmo0920** [comment]
Yep, I'd confirmed that `td-agent.bat --version` or `cd ¥path¥to¥td-agent¥bin; td-agent --version` shows td-agent version and using Fluentd version.
But I also confirmed that Td-Agent Command Prompt is forcibly exited when executing `td-agent --version` with its default working directory.

WDYT?

File: td-agent/msi/assets/td-agent.bat [hunk 1]
```diff
@@ -3,4 +3,11 @@ set TD_AGENT_TOPDIR=%~dp0\..
 set PATH="%~dp0";%PATH%
 set FLUENT_CONF=%TD_AGENT_TOPDIR%\etc\td-agent\td-agent.conf
 set FLUENT_PLUGIN=%TD_AGENT_TOPDIR%\etc\td-agent\plugin
+set TD_AGENT_VERSION=%TD_AGENT_TOPDIR%\bin\td-agent-version.rb
+for %%p in (%*) do (
+    if "%%p"=="--version" (
+        ruby "%TD_AGENT_VERSION%"
+        exit 0
```
Comment (cosmo0920) [inline td-agent/msi/assets/td-agent.bat:10]: We should remove this line because exit 0 causes unexpected Td-Agent Command Prompt shutdown.

**Review by cosmo0920 — CHANGES_REQUESTED** [review]

File: td-agent/msi/assets/td-agent.bat (hunk 1 above)
Comment (cosmo0920) [reply td-agent/msi/assets/td-agent.bat:10]: How about use `cmd /c "ruby %TD_AGENT_VERSION%"` here?

Comment (cosmo0920) [reply td-agent/msi/assets/td-agent.bat:10]: Or, how about this?

```cmd
@echo off
set TD_AGENT_TOPDIR=%~dp0\..
set PATH="%~dp0";%PATH%
set FLUENT_CONF=%TD_AGENT_TOPDIR%\etc\td-agent\td-agent.conf
set FLUENT_PLUGIN=%TD_AGENT_TOPDIR%\etc\td-agent\plugin
set TD_AGENT_VERSION=%TD_AGENT_TOPDIR%\bin\td-agent-version.rb
for %%p in (%*) do (
    if "%%p"=="--version" (
        cmd /c "ruby %TD_AGENT_VERSION%"
        goto last
    )
)
"%TD_AGENT_TOPDIR%\bin\fluentd" %*

:last
```

Comment (kenhys) [reply td-agent/msi/assets/td-agent.bat]: It looks good!

**Review by cosmo0920 — APPROVED** [review]
LGTM!

**Pull request merged**